### ISSUE TITLE
[Accessibility team] Narrator/NVDA does not announce “No data to be shown” status message when no search results are found in Events after adding namespaces

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#307]
[ADO LINK](https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/36821248)

### Actual: 
Narrator/NVDA does not announce “No data to be shown” status message when no search results are found in Events after adding namespaces

Note: This issue is also reproducing with NVDA. 
Similar issue is also reproducing in the below scenarios:
Issue 1: Create project-->Basics--> Clusters dropdown
Issue 2: This issue reproducing throughout the application where status message displayed. 

### Expected:
Narrator/NVDA should automatically announce the displayed status message when no search results are found, informing users that the search returned zero results.

### User Impact:
Screen reader users are unaware that their search has failed and returned no results. This can cause confusion and repeated attempts without understanding the system state.

### Repro Steps:

- Start Narrator/NVDA.
- Launch AKS desktop application.
- TAB to "Azure Account" button and press ENTER.
- Azure Authentication login screen will open. Sign in with your v-id.
- Afte login "Azure Account" dialog will open with "Tenant ID" and "Default subscription ID". TAB to "Add cluster from Azure" button and press enter.
- "Register AKS Cluster" dialog will open. Verify all the elements of these dialogs are accessible and MAS compliant.
- After adding the cluster home page will open.
- TAB to "All Clusters" Tab and press ENTER. 
- TAB to "More actions" button in grid and press ENTER.
- Tab to "View" button and press enter.
- TAB to "Namespaces" button and select any list item.
- Observe that Narrator/NVDA does not announce “No data to be shown” status message when no search results are found in Events after adding namespaces
